SecNodeTracker.js

```diff
@@ -366,7 +376,8 @@ class SecNode {
                                 "isValidBal": addrBal.valid,
                                 "queueDepth": count,
                                 "lastChalBlock": addrBal.lastChalBlock,
-                                "lastExecSec": local.getItem('lastExecSec')
+                                "lastExecSec": local.getItem('lastExecSec'),
+                                "SecNodeMD5": md5File.sync('SecNodeTracker.js')
```

Comment:
the MD5 can be easily gamed, just by downloading the official SecNodeTracker.js and feeding it to `SecNodeMD5` ... this is just adding to the complexity rather than solving it